src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -32,7 +32,17 @@ def record(self, organization: Organization, use_case: UseCase, string: str) ->
 
     def resolve(self, organization: Organization, use_case: UseCase, string: str) -> Optional[int]:
         # NOTE: Ignores ``use_case`` for simplicity.
-        return _STRINGS.get(string)
+        # return _STRINGS.get(string)
+
+        import random
+
+        try:
+            resolved = _STRINGS[string]
+        except KeyError:
+            # just generating a random number for now
+            resolved = random.randint(20, 100)
```

Comment:
I just added this in for now so that I could test without having to add every single string to the `_STRINGS` dictionary